sql/nondesktop_exact_mau28_by_dimensions_v1.sql

```diff
@@ -0,0 +1,49 @@
+WITH
+  inactive_days AS (
+    SELECT
+      *,
+      DATE_DIFF(submission_date, date_last_seen, DAY) AS _inactive_days,
+      DATE_DIFF(submission_date, date_last_seen_in_tier1_country, DAY) AS _inactive_days_tier1
+    FROM
+      core_clients_last_seen_v1
+  )
+
+SELECT
+  submission_date,
+  CURRENT_DATETIME() AS generated_time,
+  COUNTIF(_inactive_days < 28) AS mau,
+  COUNTIF(_inactive_days < 7) AS wau,
+  COUNTIF(_inactive_days < 1) AS dau,
+  -- You can recover MAU for all clients in tier 1 countries via a WHERE clause
+  -- and summing mau, but this considers only the country associated with the
+  -- latest observation of the client; "Tier 1 grouped" MAU is an alternative
+  -- that includes all clients seen in a tier 1 country in the 28 day window,
+  -- even if their latest ping came from an excluded country.
+  COUNTIF(_inactive_days_tier1 < 28) AS tier1_grouped_mau,
+  COUNTIF(_inactive_days_tier1 < 7) AS tier1_grouped_wau,
+  COUNTIF(_inactive_days_tier1 < 1) AS tier1_grouped_dau,
+  -- We filter down to a specific set of applications considered for 2019 goals
+  -- and provide a single clean "product" name that includes OS where necessary.
+  CASE app_name
+    WHEN 'Fennec' THEN CONCAT(app_name, ' ', os)
+    WHEN 'Focus' THEN CONCAT(app_name, ' ', os)
+    WHEN 'Zerda' THEN 'Firefox Lite'
+    ELSE app_name
+  END AS product,
```

Comment:
Because the names of these two tables are so symmetrical, my expectation would be that a `SUM(mau)` on `_by_dimensions` would give exactly the same result as `SUM(mau)` on the global table. I'm hesitant to apply any additional filtering or cleaning to the global table as they might lead to people making invalid assumptions.

I guess I see `last_seen` to `exact_mau` as the transition from generic to "cleaned" such that anything with the `exact_mau` name can be used somewhat naively and still be in line with the KPI calculations.